src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -306,7 +306,9 @@
             "PacketTxCount",
             "TxErrCount",
             "CollisionCount",
-            "OverrunCount"
+            "OverrunCount",
+            "FeatureMap",
```

Comment:
same here: featuremap is used in clusterinit, so we have to make sure it is available in zap.